### PR TITLE
add docker hub ratelimit metric

### PR DESCRIPTION
**Pull request by gamoutatsumi**
provide docker hub ratelimit as a metric

https://docs.docker.com/docker-hub/download-rate-limit/#how-can-i-check-my-current-rate

### CODE REVIEW

**Comment by whywaita** [comment]
Please add an options that user for docker hub

File: pkg/config/init.go [hunk 1]
```diff
@@ -75,6 +75,17 @@ func LoadWithDefault() Conf {
...
+	c.ProvideDockerHubMetrics = false
+	if os.Getenv(EnvProvideDockerHubMetrics) == "true" {
+		c.ProvideDockerHubMetrics = true
+	}
+
+	c.DockerHubCredential = DockerHubCredential{}
+	if os.Getenv(EnvDockerHubUsername) != "" && os.Getenv(EnvDockerHubPassword) != "" {
+		c.DockerHubCredential.Username = os.Getenv(EnvDockerHubUsername)
+		c.DockerHubCredential.Password = os.Getenv(EnvDockerHubPassword)
+	}
```
Comment (whywaita) [inline pkg/config/init.go:87]: Please logging and handling

- if set EnvProvideDockerHubMetrics is false, but set Docker credential (maybe not needed)
- if set EnvProvideDockerHubMetrics is true, but not set Docker credential (logging anonymous mode)

Comment (gamoutatsumi) [reply pkg/config/init.go:93]: thanks, reflected your reviews.

**Comment by whywaita** [comment]
Thank you!

**Pull request merged**